panel/_templates/jupyter.css

```diff
@@ -3,5 +3,13 @@
   box-sizing: border-box;
   font-family: var(--jp-ui-font-family);
   font-size: var(--jp-ui-font-size1);
-  color: var(--jp-ui-font-color1);
+  color: var(--vscode-editor-foreground, var(--jp-ui-font-color1));
+}
+
+/* Override VSCode background color */
+.cell-output-ipywidget-background:has(> .cell-output-ipywidget-background
```

Comment:
`:has()` pseudo selector is not available in Firefox yet. Given this change is related to vscode, then maybe this is not a concern.